### PR TITLE
[ETHOSN] Avoid duplicate specification of tvmc targets

### PR DESCRIPTION
**Pull request by Leo-arm**
Remove the need to specify targets as "Ethos-N -variant=Ethos-N"

Co-authored-by: Tristan O'Connor <tristan.oconnor@arm.com>

### CODE REVIEW

**Comment by Leo-arm** [comment]
@leandron please.

File: python/tvm/relay/op/contrib/bnns.py [hunk 1]
```diff
@@ -70,7 +73,7 @@ def partition_for_bnns(mod, params=None):
         ]
     )
 
-    return seq(mod)
+    return seq(mod, None)
```
Comment (leandron) [inline python/tvm/relay/op/contrib/bnns.py:76]: Is this one intentionally different from the other similar changes?

**Review by leandron — COMMENTED** [review]
There is one (probably) typo in my comment below. Apart from that, could you just double check whether there is any tests to be improved so that `-variant` is not used in tests, and we validate the behaviour introduced here?

File: python/tvm/relay/op/contrib/bnns.py (hunk 1 above)
Comment (Leo-arm) [reply python/tvm/relay/op/contrib/bnns.py]: No, my bad, and fixed.

**Comment by jroesch** [comment]
@Leo-arm can we update this one?

**Comment by Leo-arm** [comment]
@jroesch Sorry, missed your comment completely. Yes, will do hopefully this week.

**Comment by Leo-arm** [comment]
Not entirely happy with this one as there are a lot of changes for a relatively simple ask. I'll come back to this when I have a better solution.

**Pull request closed without merging**